examples/roitberg/pyanitools.py

```diff
@@ -0,0 +1,126 @@
+# Written by Roman Zubatyuk and Justin S. Smith
+import h5py
+import numpy as np
+import platform
+import os
+
+PY_VERSION = int(platform.python_version().split('.')[0]) > 3
+
+
+class datapacker(object):
```

Comment:
Ok, let's merge without forking the code for now. We can always change later as needed.